helm-charts/strimzi-kafka-operator/templates/020-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -63,6 +63,8 @@ rules:
     # The cluster operator runs the KafkaBridgeAssemblyOperator, which needs to access BridgeMaker resources
   - kafkabridges
   - kafkabridges/status
+  - kafkamirrormaker2s
```

Comment:
Can you add a comment (like above for the bridge operator) explaining why access is necessary. I know it's obvious, but it's part of a wider effort to be more transparent about why the operator needs the access it does. We often get questions from users/admins so if we can get to the point where it's exhaustively documented then it makes all these permissions less daunting.

Reply:
Done. Sorry I missed that one.